css/index.css

```diff
@@ -1,3 +1,154 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+* {
+	box-sizing: border-box;
+	font-family: "Titillium Web", sans-serif;
+}
+body {
+	margin: 2vh 8vh;
+}
+nav {
+	display: flex;
+	justify-content: space-between;
+}
+nav a {
+	padding: 3vh;
+	text-decoration: none;
+	color: grey;
+}
+header {
+	display: flex;
```

Comment:
Research em and rem units and how they can be useful for font size, padding, and margin in responsive design.